docs/examples/ex_secret_types.py

```diff
@@ -5,12 +5,15 @@ class SimpleModel(BaseModel):
     password_bytes: SecretBytes
 
 sm = SimpleModel(password='IAmSensitive', password_bytes=b'IAmSensitiveBytes')
+
+# Standard access methods will not display the secret
 print(sm)
+print(str(sm.password))
```

Comment:
```suggestion
print(repr(sm.password))
```
`print(str(x))` is the same as `print(x)`